frontend/src/components/SubResult.vue

```diff
@@ -1,15 +1,35 @@
 <script setup lang="ts">
 import BetaversionLabel from '@/components/betaversion-label.vue'
+import { computed } from 'vue'
 
 interface Props {
-  topic: string | undefined
+  category: string | undefined
   labels: { category: string; assigned_labels: string | undefined; }[] | undefined
   title: string
   conclusion: string
 }
 
-defineProps<Props>()
+  const props = defineProps<Props>()
 
+
+// Function to check if there are any valid labels in a category
+function hasValidLabelsForCategory(assigned_labels: string[]) {
+  return assigned_labels.some(label => label !== 'nader te bepalen' && label !== 'niet van toepassing')
```

Comment:
```suggestion
  return assigned_labels.some(label => label !== 'nader te bepalen' )
```

I would remove the 'niet van toepassing' because this is kinda of the same as geen 'label' which is a relevant label.